pom.xml

```diff
@@ -1172,6 +1173,19 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>travis_main</id>
```

Comment:
This profile is used for skipping e2e tests, that take long. From `main` it is not clear what it does, is travis_e2e_skip better?